int32 numObj
float64[] meanArray
float64[] eivecArray
float64[] pcArray
